mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -44,7 +55,16 @@ impl BridgeSettings {
 
         config.merge(File::from_str(DEFAULTS, FileFormat::Json))?;
         config.merge(File::from(path.as_ref()))?;
-        config.merge(Environment::new())?;
+        config.merge(Environment::new().separator(ENV_VAR_SEPARATOR))?;
```

Comment:
Added to try to be in line with https://github.com/Azure/iotedge/blob/master/doc/EnvironmentVariables.md Vadim, I know you recently worked on env vars, so if there is a style I should change to, let me know!